index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+$db = new PDO('mysql:host=db; dbname=ukgamesfunded', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare("SELECT `title`, `year` , `theme` , `genre` , `rating`  FROM `game`");
+
+$query->execute();
+
+$result = $query->fetchAll();
+
+echo '<h2>UK Games Funded</h2>';
```

Comment:
This line can be moved down into the HTML (so after the closing `?>` PHP tag), just because there is nothing dynamic going on, so it would be slightly quicker to just have 

`<h2>UK Games Funded</h2>` instead of echoing it from the PHP